skyportal/models/observation_plan.py

```diff
@@ -71,6 +71,17 @@ class DefaultObservationPlanRequest(Base):
         overlaps='groups',
     )
 
+    default_plan_name = sa.Column(
+        sa.String, unique=True, nullable=False, doc='Default plan name'
+    )
```

Comment:
I'm not sure this unique constraint is doing what you think it should. 
Are there not going to be any duplicate default plans at any time on the DB?

Reply:
I dont want any with the same name at least. :)

Reply:
Is it a global constraint? Not specific to each user or each localization or something like that?

Reply:
Yeah I want people to make different names for their plans.